docs/README.md

```diff
@@ -450,6 +450,7 @@ This block renders a filter selector for the fetched results.
 | `fullWidthOnMobile`      | `boolean` | Whether the `filter-navigator.v3` will be rendered on mobile using the full screen width (`true`) or not (`false`).     | `false`       |
 | `navigationTypeOnMobile` | `enum`    | Defines how mobile users should navigate on the filter selector component. The possible values are `page` (only one list of options can be seen at a time) or `collapsible` (all lists of options can be seen simultaneously).                                    | `page`        |
 | `updateOnFilterSelectionOnMobile` | `boolean` | Whether the search results on mobile should be updated according to filter selection (`true`) or not (`false`). This prop only works if the `preventRouteChange` prop is declared as `true`.         | `false`       |
+| `drawerDirectionMobile` | `Enum` | Whether the search filters on mobile opens to the left (`drawerLeft`) or to the right (`drawerRight`) | `drawerLeft` |
```

Comment:
it's just an opinion, but since the name of the prop already indicates that it's a direction, I think the options could simply be `left` and `right`

Reply:
I think it was to match the "Animation" from store-components props. I can change it if you want.

Reply:
you can keep like that, as I said it was just an opinion 😄